webpack/redux/actions/RedHatRepositories/helpers.js

```diff
@@ -14,25 +14,23 @@ const recommendedRepositoriesRHEL = [
   'rhel-8-for-x86_64-baseos-kickstart',
   'rhel-8-for-x86_64-appstream-rpms',
   'rhel-8-for-x86_64-appstream-kickstart',
+  'rhel-8-for-x86_64-baseos-eus-rpms',
+  'rhel-8-for-x86_64-appstream-eus-rpms',
   'rhel-7-server-rpms',
   'rhel-7-server-optional-rpms',
   'rhel-7-server-extras-rpms',
-  'rhel-7-server-eus-rpms',
   'rhel-7-server-kickstart',
-  'rhel-6-server-els-rpms',
-  'rhel-6-server-kickstart',
 ];
 
 const recommendedRepositoriesSatTools = [
-  'satellite-tools-6.10-for-rhel-8-x86_64-rpms',
-  'rhel-7-server-satellite-tools-6.10-rpms',
-  'rhel-6-server-els-satellite-tools-6.10-rpms',
-  'rhel-7-server-satellite-maintenance-6-rpms',
+  'satellite-client-7-for-rhel-8-x86_64-rpms',
+  'rhel-7-server-satellite-client-7-rpms',
+  'rhel-7-server-satellite-maintenance-7.0-rpms',
 ];
 
 const recommendedRepositoriesMisc = [
   'rhel-server-rhscl-7-rpms',
-  'rhel-7-server-satellite-capsule-6.10-rpms',
+  'rhel-7-server-satellite-capsule-7.0-rpms',
```

Comment:
lets hold off on merging this for a few days, some of this may change

Reply:
how prescient @jlsherrill :smile: